stories/documentation/users/select/user-select.stories.ts

```diff
@@ -12,7 +12,9 @@ import { Meta, StoryFn, applicationConfig } from '@storybook/angular';
 	imports: [LuUserSelectModule, FormsModule],
 })
 class UserSelectStory {
-	@Input() public model;
+	public model;
```

Comment:
model no longer an input? (Also please type it as unknown or explicit any)

Reply:
before the pull request, the model could not be modified from the story by the user 
as it's the value selected from the input, I guess it doesn't really make sense to allow an input on it (for the stories)